openstack/utils/templates/_hosts.tpl

```diff
@@ -46,7 +52,13 @@ postgresql+psycopg2://{{$user}}:{{$password | urlquery}}@{{.Chart.Name}}-postgre
 
 {{- define "db_url_mysql" }}
     {{- if kindIs "map" . }}
-        {{- tuple . (coalesce .Values.dbName .Values.db_name) (coalesce .Values.dbUser .Values.global.dbUser "root") (coalesce .Values.dbPassword .Values.global.dbPassword .Values.mariadb.root_password | required ".Values.mariadb.root_password is required!") .Values.mariadb.name | include "db_url_mysql" }}
+        {{- if and .Values.mariadb.users .Values.mariadb.databases }}
+            {{- $db := first .Values.mariadb.databases }}
+            {{- $user := get .Values.mariadb.users $db | required (print "User with key " $db " required") }}
```

Comment:
Can we align the help-text with the one above? `"User with key of the first database in .Values.mariadb.databases (" $db ") required"`

Reply:
I am not quite clear what you mean with align?

The second one is simply a missing field, and we usually do something like `.xyz is required/missing`, which I think is sufficient.
But the first one has to explain the interdependency of the values. I tried to reformulate it... hopefully that fits the request.

Reply:
ok, let me explain `"User with key of the first database in .Values.mariadb.databases (" $db ") required"` in the first `get .Values.mariadb.users $db` on line 28 is more verbose than `"User with key " $db " required"` on line 57. Both take the user automatically after looking at the first value in `.databases`. I just wanted the `"User with key " $db " required"` to be `"User with key of the first database in .Values.mariadb.databases (" $db ") required"`, because to me the additional explanation of how we come to needing that specific user is a good hint for understanding it all.

Reply:
Got it, they are now the same.